api-js/src/affiliation/mutations/update-user-role.js

```diff
@@ -169,7 +175,9 @@ export const updateUserRole = new mutationWithClientMutationId({
           `User: ${userKey} attempted to lower user: ${requestedUser._key} from ${affiliation.permission} to: user.`,
         )
         throw new Error(
-          i18n._(t`Unable to update users role. Please try again.`),
+          i18n._(
+            t`Permission Denied: Please contact organization admin for help with updating users roles.`,
```

Comment:
Same

Reply:
This is fine

Reply:
The original change, that is

Reply:
Except it should say "for help with updating user roles"

Reply:
MOAR COMMENTS